docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -852,3 +852,32 @@ and prepare this functionality for the upcoming Kotlin 2.0 release. From now on:
 
 Starting with Kotlin 1.9.20, you need to explicitly make a cast to and from the corresponding C and Objective-C forward
 declarations. Also, it's now only possible to import forward declarations by using special packages.
+
+## Incorrect version of iOS framework after Kotlin upgrade
+
+**What's the issue?**
+
+If you use direct integration through the `embedAndSignAppleFrameworkForXcode` task, which connects the iOS framework
```

Comment:
I think the "What's the issue?" question should be answered as soon as possible to help users understand sooner that it is the issue they encountered or not.

It might help to restructure these paragraphs:
* The changes in Kotlin code are not reflected in the iOS app in Xcode when you use direct integration <...>.
* This can happen when you upgrade Kotlin <...> then make changes <...>: Xcode may incorrectly <...>